docker/Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM ubuntu:jammy
+
+RUN apt-get update && apt-get install -y wget
+EXPOSE 9050
+
+COPY ./setup.sh /setup.sh
+# RUN  wget https://sh.iroh.computer/setup.sh && chmod 700 setup.sh
+RUN chmod 700 setup.sh
+RUN ./setup.sh init
+
+ENV PATH="${PATH}:~/.iroh/bin"
+
+CMD /bin/sh -c "./setup.sh start && tail -n 100 -f ~/.iroh/log/*"
```

Comment:
while this does technically "put iroh on docker", this isn't the right way to go about it. Instead we should have a dockerfile for each service (`iroh-store`, `iroh-p2p`, `iroh-gateway`), and those dockerfiles should use [multi-stage builds](https://docs.docker.com/build/building/multi-stage/) to build production rust binaries & run them on minimal distribution environments.

Given how involved the task is, it might make sense to have someone on the iroh team put together those Dockerfiles, and maybe rebase this PR to build a `docker-compose.yml` example on top of those